docs/setup_vscode.md

```diff
@@ -484,189 +449,163 @@ To configure input redirection, edit `launch.json`.
+<div class="primer-spec-callout warning" markdown="1">
+If you have trouble viewing the contents of a container like this screenshot, see [Pretty Printing STL Containers with `gdb`](setup_gdb.html#pretty-printing-stl-containers-with-gdb).
 
 <img src="images/vscode140.png" width="480px" />
+</div>
 
+### Step into
+Click "Step Into".  The cursor enters the `sum()` function.
 
-## macOS Visual Debugger Issues
-On older versions of macOS, VS Code, or the C/C++ extension, you may experience issues with the visual debugger. Please try the following steps to troubleshoot your debugger:
+<img src="images/vscode115.png" width="768px" />
 
-1. Restart your computer
-2. Update macOS, VS Code, and the C/C++ extension.  This tutorial has been tested as of macOS Big Sur 11.1, VS Code 1.52.1, and C/C++ Extension version 1.1.3.
-3. Try using CodeLLDB's debugger instead (see below)
+<img src="images/vscode120.png" width="768px" />
 
-### Install CodeLLDB
-{: .primer-spec-toc-ignore }
-Another option in troubleshooting your visual debugger is to replace the one that comes with the C/C++ extension you set up in the beginning of this tutorial.
+### Step out
+Click "Step Out".  The `sum()` function completes, and the program pauses again.
 
-We **do not** recommend you uninstall the C/C++ extension because it provides syntax highlighting and other various niceties; instead, you can have both plugins installed concurrently.
+<img src="images/vscode125.png" width="768px" />
 
-<img src="images/vscode021.png" width="768px" />
+<img src="images/vscode130.png" width="768px" />
 
-After you've installed the plugin, you'll need to update your `launch.json` to specify that you want to use the CodeLLDB plugin instead. Replace your `launch.json` with the following:
+### Continue
+Press "Continue" to run the program to the next breakpoint, or the end, whichever comes first.
 
-```json
-{
-    "version": "0.2.0",
-    "configurations": [
-        {
-            "name": "(lldb) Launch",
-            "type": "lldb",
-            "request": "launch",
-            "program": "${workspaceFolder}/stats_tests.exe",
-            "args": [],
-            "cwd": "${workspaceFolder}"
-        }
-    ]
-}
-```
+<img src="images/vscode135.png" width="768px" />
 
-If this method works for you, feel free to continue using the CodeLLDB debugger!
+## Troubleshooting
+This section is for common problems and solutions.
 
-Note that if you are using CodeLLDB, [the above section describing `launch.json`
-configuration](#run) is no longer accurate. Instead, refer to the below
-subsections.
+### Intellisense and C++11
```

Comment:
Change header to C++17 for consistency with the suggested modification? Or just something generic like "Intellisense C++ Standard"?